### PR TITLE
Nixify [devShell only] load test

### PR DESCRIPTION
**Pull request by soumyajit2304**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

**Review by srid — CHANGES_REQUESTED** [review]
Gave offline feedback to @soumyajit2304 about refactoring this. 

(Will pair when in office, if necessary)

**Comment by soumyajit2304** [comment]
Update the PR with the required changes.

**Review by srid — COMMENTED** [review]
@soumyajit2304 Could you update `backend/README.md` with exact instructions to follow for running the load test in devShell, so that I can test your PR locally (to make sure that it is reproducible)?

**Comment by soumyajit2304** [comment]
I have updated the load-test Readme `Backend/load-test/README.md`.

File: Backend/load-test/README.md [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
+, run-svc -d
+, run-mobility-stack-dev
 ```
 
 ### To run the load-test
 
+You can run the load test by
 ```
-, run-load-test
+ , run-load-test-nix
```
Comment (srid) [inline Backend/load-test/README.md:13]: Is this correct?

`, run-load-test-nix` just runs `nix run` which fails for me:

I can run the load-test on my machine:

image

**Comment by srid** [comment]
@soumyajit2304 Now I'm seeing this - one process in the load-test group crashed:

image

**Comment by srid** [comment]
Alright, that worked after I deleted the output folder. But now I get this. However this is not Nix's fault, so I don't think we should block this PR.

image

**Comment by soumyajit2304** [comment]
> Alright, that worked after I deleted the output folder. But now I get this. However this is not Nix's fault, so I don't think we should block this PR.
> 
> image

Yes this is fine and expected as entry are already made in the table. 
So next process will run once this goes to completed which are dependent on this

**Comment by srid** [comment]
For reference, we are using derivations from:

- https://github.com/NixOS/nixpkgs/pull/185087 
- https://github.com/NixOS/nixpkgs/pull/199746 

Neither of these PRs were merged and they have been sitting there for a while.

**Review by srid — APPROVED** [review]

**Pull request merged**